src/utilities/filetypes/RunOptions.hpp

```diff
@@ -96,6 +100,10 @@ class UTILITIES_API RunOptions
   bool setCustomOutputAdapter(const CustomOutputAdapter& adapter);
   void resetCustomOutputAdapter();
 
+  std::string forwardTranslateOptions() const;
```

Comment:
forwardTranslateOptions cannot be stored as a std::string and naively passed on to the workflow gem. Or there will be no type checking: no one is currently checking whether the options are bools or not.

```
"run_options": {
     "debug": true,
     "ft_options": {
       "no_html_output": "POTATOE POTATO",
     },
```

https://github.com/NREL/OpenStudio-workflow-gem/pull/137/files#diff-25861076be80d19ea996e98aedc50b0ee85414ca645ddca3976f906b8e63398cR368-R383

So you end up with `ft_opts['no_html_output'] = "POTATOE POTATO" `.

Later on it's passed as a C++ call to the ForwardTranslator method `ft.method(ft_method).call(opt_flag)` here: https://github.com/NREL/OpenStudio-workflow-gem/pull/137/files#diff-4f1b9fe57e98b6d54dbe147500735448da91a8fe5f711bf0f0f1fb88e8734e78R117

This will Throw: 

ft.setExcludeHTMLOutput("POTATOE POTATO") => Throw: wrong type, expected bool but got string